docs/src/main/asciidoc/building-native-image-guide.adoc

```diff
@@ -209,6 +209,13 @@ duration can be changed using the `quarkus.test.native-image-wait-time` system p
 to 300 seconds, use: `./mvnw verify -Pnative -Dquarkus.test.native-image-wait-time=300`.
 ====
 
+By default, native tests runs using the `prod` profile.
+This can be overridden using the `quarkus.test.native-image-profile` property.
+For example, in your `application.properties` file, add: `quarkus.test.native-image-profile=test`.
+Alternatively, you can run your tests with: `./mvnw verify -Pnative -Dquarkus.test.native-image-profile=test`.
+However, don't forget that when the native executable is built the `prod` profile is enabled.
```

Comment:
I find this sentence confusing. Are you saying that your tests will run with the test profile but pointing to the image that was built as prod.
Or that your image will be build with test and that should be compatible with the "prod" environment you have in mind.

My fear is that this is really inside baseball here and that's typically this is written for the implementer to feel happy that he has covered all cases than in the spirit of helping the user. If we see different use cases, we should enunciate them better so people could connect to the problem + solution.

This is about improvement so if we cannot make it I won't die for it.